torch/_torch_docs.py

```diff
@@ -6656,6 +6656,30 @@ def merge_dicts(*dicts):
     tensor([ -24.8863, -771.4742])
 """.format(**common_args))
 
+add_docstr(torch.positive,
+           r"""
+positive(input, *, out=None) -> Tensor
+
+Returns a new tensor with the positive of the elements of :attr:`input`.
+
+.. math::
+    \text{out} = +1 \times \text{input}
+""" + r"""
```

Comment:
Let's clarify that for all native PyTorch dtypes this operation is equivalent to cloning the tensor by adding a sentence here.

Reply:
If we implement positive() as `return self;` then the above comment is no longer correct. (See comment on returning self above). 

With that change the function can be documented as "Returns :attr:\`input\`. Conceptually this is equivalent to" and then the mathematical expression.